### ISSUE TITLE
Upload to pypi

### ISSUE THREAD

**Issue by andrewgsavage** [hgrecco/pint-pandas#8]
Is there anything else we'd like to do before uploading it to pypi?

**Comment by hgrecco** [comment]
I completely missed this issue. I think I can upload, I just have a question regarding the version number. How about assigning the same major and minor version of the targered pint version?

**Comment by andrewgsavage** [comment]
That makes sense.

**Comment by oogieoogieful** [comment]
Hello, will this be upload to pypi, so that I can install with pip?

**Comment by awa5114** [comment]
@andrewsavage perhaps make it compatible with the newest pandas (0.25)?

**Comment by dschien** [comment]
@hgrecco Any updates on this? It would be great to have this on pypi. In my opinion it makes sense to upload as is.

**Comment by dschien** [comment]
Hi @hgrecco, @andrewgsavage 

I have some collaborators that struggle with installing from git. 
So, I just uploaded a fork to pypi: https://pypi.org/project/Pint-Pandas-Fork/0.1.dev0/

I tried to be as polite as possible, broadly marking this as a fork and acknowleding @hgrecco.
I will take it off pypi immediately, if anyone is offended. 

This also relates to https://github.com/hgrecco/pint-pandas/issues/22 I suppose.
